proto/mixer.proto

```diff
@@ -567,6 +567,43 @@ message SearchStatVarResponse {
   repeated EntityInfo stat_var_groups = 2;
 }
 
+message StatVarSummary {
+  message Place {
+      string dcid = 1;
+      string name = 2;
+      int64 person_count = 3;
+    }
+    message PlaceTypeSummary {
+      // Number of places with stats for this stat var
+      int64 num_places = 1;
+      // List of example places that have stats for this stat var
+      repeated Place top_places = 2;
+    }
+    // Key is type of place
+    map<string, PlaceTypeSummary> place_type_summary = 1;
+}
+
+message GetStatVarSummaryRequest {
+  // A list of stat var dcids
+  repeated string stat_vars = 1;
+}
+
+message GetStatVarSummaryResponse {
+  map<string, StatVarSummary> stat_var_summary = 1;
+}
+
+message FilterStatVarRequest {
```

Comment:
Maybe we can make the API "CheckPlaceStats"? So the request is the same, but response is the raw data of mapping of place -> stat_var -> boolean. This way, user can use the response with more flexibility.

Also this is very similar to /place/stat-vars, where request have N places, and returns all the stat vars. Here, instead of return all stat vars, we check if given stat vars are valid (which in theory can be done if user just convert /place/stat-vars response to a Set). Maybe we can make this an extension of that, like /place/stat-vars/existence?

Reply:
Add a message comment on what it does?

Reply:
Since we just want the existence of the stat var for any of the list of places in the request, maybe response of stat_var -> boolean makes more sense?

Reply:
That could be an option. It would be nice to see how clients will use the response. For example, client may want, the union of the stat vars, the intersection of the stat vars, or just the count. Can you try to see what it looks like if the response is a list / map[bool] for each case?

With that, I wonder is it better to key by place or stat var id. If the first level key is place id, could we make this API part of the /place/stat-var API with additional filter (stat var ids).

Reply:
removing this endpoint from this PR as it requires some additional thought and design